lib/pbench/client/oidc_admin.py

```diff
@@ -0,0 +1,150 @@
+from http import HTTPStatus
+import json
+import os
+
+import requests
+
+from pbench.server.auth import Connection
+
+
+class OIDCAdmin(Connection):
+    OIDC_REALM = "pbench-server"
+    ADMIN_USERNAME = os.getenv("ADMIN_USERNAME", "admin")
+    ADMIN_PASSWORD = os.getenv("ADMIN_PASSWORD", "admin")
```

Comment:
Should the realm come from an environment variable (perhaps with `"pbench-server"` as a default) like the Admin username/password?